pkg/provider/data_source_cluster.go

```diff
@@ -371,6 +371,10 @@ func (c *clusterDataSource) Schema(ctx context.Context, req datasource.SchemaReq
 					},
 				},
 			},
+			"pause": schema.BoolAttribute{
+				Description: "Pause cluster. If true it will put the cluster on pause and set the phase as paused, if false it will resume the cluster and set the phase as healthy",
```

Comment:
we can also add : 
- Pausing a cluster allows you to save on compute costs without losing data or cluster configuration settings. 
- While paused, clusters aren't upgraded or patched, but changes are applied when the cluster resumes. Pausing a high availability or Postgres Distributed cluster shuts down all cluster nodes.

Reply:
I changed the description on pg and pgd have a look